apps/blocklist/tests/test_views.py

```diff
@@ -13,7 +13,7 @@
 from amo.urlresolvers import reverse
 from blocklist.models import (BlocklistApp, BlocklistCA, BlocklistDetail,
                               BlocklistGfx, BlocklistItem, BlocklistPlugin,
-                              BlocklistPref)
+                              BlocklistPref, BlocklistIssuerCert)
```

Comment:
We order the imports alphabetically, so this should go on he line above before `BlocklistPlugin`, thanks!